src/main/database/table-definitions.ts

```diff
@@ -1,12 +1,12 @@
 export const StationEvents: string = `
-      bibId INTEGER DEFAULT (0),
-      stationId INTEGER,
+      bibId INTEGER DEFAULT (0), // TODO: Index
```

Comment:
Keep in mind that while indexes speed up SELECTs, they slow down INSERT/UPDATEs. So, review the queries that use these in the WHERE (or JOIN..ON) clause. If those queries need to be faster, add an index. But use caution when adding them to tables with high-frequency INSERT/UPDATEs.